lumeer-storage/lumeer-storage-mongodb/src/main/java/io/lumeer/storage/mongodb/dao/project/MongoLinkInstanceDao.java

```diff
@@ -35,7 +36,7 @@
    public void createLinkInstanceRepository(Project project) {
       database.createCollection(databaseCollectionName(project));
 
-      MongoCollection<LinkInstance> projectCollection = databaseCollection();
+      MongoCollection<Document> projectCollection = database.getCollection(databaseCollectionName(project));
```

Comment:
Why this change?

Reply:
It's same as in other Dao-s. It's cause test exception, because when we are creating repository on Project creation, we don't have set `Project` property here (but it's correct behavior).